<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Lbl_Question">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>600</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard ExtraBold</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>햄버거를 골라주세요.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_bulgogiBuger">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>790</y>
     <width>500</width>
     <height>240</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>100</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>132</red>
         <green>64</green>
         <blue>19</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>72</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(132, 64, 19);
color:rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>불고기 버거</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_back">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>300</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_cheeseBuger">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>790</y>
     <width>500</width>
     <height>240</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>72</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 217, 0);</string>
   </property>
   <property name="text">
    <string>치즈 버거</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_chickenBuger">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>790</y>
     <width>500</width>
     <height>240</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>72</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 140, 0);
color:rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>치킨 버거</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_next">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>40</y>
     <width>300</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>다음 단계로</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_bulgogiBugerImg">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/bulgogi.png</normaloff>../img/buger/bulgogi.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>700</width>
     <height>700</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_cheeseBugerImg">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>280</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/cheese.png</normaloff>../img/buger/cheese.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>700</width>
     <height>700</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_chickenBugerImg">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>280</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/buger/chicken.png</normaloff>../img/buger/chicken.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>700</width>
     <height>700</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_bulgogi">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>72</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_cheese">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>240</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>72</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_chicken">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>240</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pretendard Variable</family>
     <pointsize>72</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
